src/components/Icons/Icon.mdx

```diff
@@ -120,22 +120,24 @@ fill the full width of the parent element.
 ## All Icons
 
 The following icons are for generic purposes. All the available icon names can
-be found in the `IconNames` type. They are: `"accessibilityFull"`,
```

Comment:
`accessibilityFull` was removed?

Reply:
I was working on a way to generate this list dynamically.  I couldn't get the MDX back tick formatting to work, so I gave up on that, but I must have accidentally dropped `accessibilityFull` in the process.